swap_meet/decor.py

```diff
@@ -1,2 +1,12 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+class Decor(Item):
+    def __init__(self, id = None, width = 0, length = 0, condition = 0, age = None):
+        super().__init__(id, condition, age)
+        self.width = width
+        self.length = length
+    def get_category(self):
+        return "Decor"
+    def __str__(self):
+        return (f"An object of type {self.get_category()} with id {self.id}. "
+                f"It takes up a {self.width} by {self.length} sized space."
```

Comment:
Notice that the first sentence in this string looks pretty similar to what you wrote  for the parent class `__str__` method (line 13 of `item.py`). 

If you refactor the parent class `__str__` method like I suggest below, you can leverage that shared functionality to reduce duplication here. 

```suggestion
        return f"super()__str__() It takes up a {self.width} by {self.length} sized space."
```